xinference/model/llm/pytorch/core.py

```diff
@@ -241,7 +245,10 @@ def generator_wrapper(
         else:
             device = self._pytorch_model_config.get("device", "cuda")
         if not stream:
-            if "falcon" in self.model_family.model_name:
+            if (
```

Comment:
"falcon" in self.model_family.model_name: already include falcon and falcon-instruct.
No need to change this command.